envFiles/.env.devcontainer

```
@@ -28,6 +28,7 @@ API_IS_GRAPHIQL=true
 API_IS_PINO_PRETTY=true
 API_JWT_EXPIRES_IN=900000
 API_REFRESH_TOKEN_EXPIRES_IN=604800000
+API_PASSWORD_RESET_TOKEN_HMAC_SECRET=b4896453be722d5ca94058a73f52b31c75980b485fa6d74d91f417a8059d8731
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_PASSWORD_RESET_TOKEN_HMAC_SECRET key should go before the API_REFRESH_TOKEN_EXPIRES_IN key

(UnorderedKey)